src/platform/imx8m/include/platform/lib/dma.h

```diff
@@ -21,6 +21,17 @@
 #define dma_chan_irq(dma, chan) dma_irq(dma)
 #define dma_chan_irq_name(dma, chan) dma_irq_name(dma)
 
+/* SDMA2 specific data */
+
+/* Interrupts must be set up interestingly -- shift them all by 32 like
+ * on the other platforms.
+ */
+
+#define SDMA2_IRQ	7 /* TODO What? */
+#define SDMA2_IRQ_NAME	"irqstr2" /* TODO find the correct one */
+
```

Comment:
Whe should remove the TODOs here. If we are sure the numbers are correct.

Reply:
I am not 100% certain. I may also end up using irqstr_get_sof_int like on the other platforms (perhaps my calculations were in fact incorrect).